gensrc/proto/olap_file.proto

```diff
@@ -175,6 +183,8 @@ message RowsetMetaPB {
     optional uint32 max_compact_input_rowset_id = 61;
     // global transaction id
     optional int64 gtid = 62;
+    // insert mode
+    optional InsertMode insert_mode = 63 [default = UPSERT_MODE];
```

Comment:
why not put this in 
    // some txn semantic information bind to this rowset
    optional RowsetTxnMetaPB txn_meta = 55;

Reply:
`txn_meta` was only created in update case
![image](https://github.com/StarRocks/starrocks/assets/7404824/55064487-e15e-4cb8-b1d0-d28e18799d0b)